src/player.cpp

```diff
@@ -1382,7 +1382,7 @@ void player::update_bodytemp()
         if( i == bp_mouth || i == bp_foot_r || i == bp_foot_l || i == bp_hand_r || i == bp_hand_l ) {
             // Handle the frostbite timer
             // Need temps in F, windPower already in mph
-            int wetness_percentage = 100 * body_wetness[i] / mDrenchEffect.at( static_cast<body_part>( i ) ); // 0 - 100
+            int wetness_percentage = 100 * body_wetness[i] / drench_capacity[static_cast<body_part>( i )]; // 0 - 100
```

Comment:
You should remove all the `static_cast<body_part>( i )` when accessing the `std::array`s. It's not needed (was needed for the maps, which had keys of `body_part` type) anymore and it is actually pointless as the `operator[]` of `std::array` requires a `size_t` (or similar).